src/platform/user/tests/profile/utilities/index.unit.spec.js

```diff
@@ -84,7 +84,9 @@ describe('Profile utilities', () => {
         },
       });
 
-      expect(mappedData.status).to.equal(data.attributes.va_profile.status);
+      expect(mappedData.veteranStatus.status).to.equal(
```

Comment:
This test should not change. The `va_profile.status` value from the payload is still mapped directly the `status`